examples/charm++/TRAM/Makefile

```diff
@@ -1,10 +1,6 @@
 -include ../../../include/conv-mach-opt.mak
 
-DIRS = \
-  randomAccessGroup \
-  randomAccessArray \
-  aggregateRandomAccessGroup \
-  aggregateRandomAccessArray \
+DIRS = aggregateRandomAccessArray
```

Comment:
Why delete the non-aggregate example, and not refactor it to match the new API?

Reply:
The non-aggregate examples were there originally to test the non-aggregate API, which isn't supported any more. I could refactor them, but they would end up being near-identical to the the aggregate test.

Reply:
The discontinued support for the non-aggregate API should be documented, as the non-aggregate API is the one currently discussed in the documentation (there is no mention of the `[aggregate]` tag there by the way).